platform/wab/src/wab/client/components/sidebar/LeftComponentsPanel.tsx

```diff
@@ -118,12 +108,32 @@ const LeftComponentsPanel = observer(function LeftComponentsPanel() {
     studioCtx.appCtx.appConfig
   );
 
-  const onAddComponent: AddComponent = (folderName) => {
-    return async () => {
+  const onDeleteFolder = React.useCallback(
+    async (folder: ComponentFolder) => {},
+    [studioCtx]
+  );
+
+  const onFolderRenamed = React.useCallback(
+    async (folder: ComponentFolder, newName: string) => {},
+    [studioCtx]
+  );
+
+  const onAddComponent = React.useCallback(
+    async (folderName?: string) => {
       const folderPath = getFolderWithSlash(folderName);
       await studioCtx.siteOps().createFrameForNewComponent(folderPath);
-    };
-  };
+    },
+    [studioCtx]
+  );
+
+  const actions: ComponentFolderActions = React.useMemo(
```

Comment:
In this commit, I grouped folder actions instead of passing them through the ComponentFolder individually. If this is preferred, I can update the token panel and nav dropdown to match. If not, I can switch this code to match the others.

Reply:
I went ahead and aligned the folder action usage between components/tokens/arenas.